[MM-34]: Added the test cases for adding assignee in issue on jira through slash command on MM. (#25)
Brightscout/mattermost-test-management: data/test-cases/plugins/jira/jira-server/jira-assign/Assign_user.md

---
# (Required) Ensure all values are filled up

> name: "Assignee a user to a jira issue through slash command."

Reviewer: ```suggestion
name: "Assign a Jira issue to a user through slash command.."
```

status: Active
priority: Normal
folder: Jira-assign
authors: "@arush-vashishtha"
team_ownership: []
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

> 1. Add a issue key in the following slash command `/jira assign [issue key] [assignee]` in the channel on MM.

Reviewer: This can be rephrased as "Run the /jira assign <issue key> <assignee> and add the desired issue key in the slash command."

> 2. Add the MM user name in the [assignee] in above command followed by a `@` when the user's MM account is connected to jira account.

Reviewer: ```suggestion
2. Add the MM user name in the <assignee> in above command followed by a `@` when the user's MM account is connected to jira account.
```

Reviewer: Fix everywhere

> 2. Add the MM user name in the [assignee] in above command followed by a `@` when the user's MM account is connected to jira account.

Reviewer: ```suggestion
2. Add the MM username in the [assignee] in above command followed by a `@` when the user's MM account is connected to jira account.
```

Reviewer: fix everywhere

**Step 2**

> 1. Add a issue key in the following slash command `/jira assign [issue key] [assignee]` in the channel on MM.

Reviewer: Same as above

2. Add the MM user name in the [assignee] in above command not followed by a `@` when the user's MM account is connected or disconnected to jira account.

**Step 3**

1. Add a issue key in the following slash command `/jira assign [issue key] [assignee]` in the channel on MM.

> 2. Add the MM user name in the [assignee] in above command when the user's MM account is disconnected to jira account.

Reviewer: Is this only specific for disconnected user, whether we start with '@' or without it?

Author: yes in both the cases it'll show the error, although the error are different. so should i update the test cases? for the different errors.
